src/components/material/ReviewInfomedia.tsx

```diff
@@ -61,7 +98,8 @@ const ReviewInfomedia: React.FC<ReviewInfomediaProps> = ({ review }) => {
         />
       )}
       {/* eslint-enable react/no-danger */}
-      {review.origin && (
+      {/* We want to make sure a link can be made out of the review origin (includes a ".") */}
+      {review.origin && review.origin.includes(".") && (
```

Comment:
I don't think this is strict enough.
We need this in other places, eg. in the proxy url functionality. So we could create a function that checks if the string is a valid url.
Something like:
```typescript 
const isValidHttpUrl = (text: string) => {
  try {
    const url = new URL(text);
    return url.protocol === 'http:' || url.protocol === 'https:';
  } catch (err) {
    return false;
  }
}